hermes-common/src/main/java/pl/allegro/tech/hermes/common/admin/zookeeper/ZookeeperAdminTool.java

```diff
@@ -47,10 +35,8 @@ public void retransmit(SubscriptionName subscriptionName) {
     private void executeAdminOperation(SubscriptionName subscriptionName, String name) throws Exception {
         String path = zookeeperPaths.adminOperationPath(name);
 
-        String createdPath = curatorFramework.create()
-                .withMode(CreateMode.EPHEMERAL_SEQUENTIAL)
+        curatorFramework.create()
+                .withMode(CreateMode.CONTAINER)
```

Comment:
```suggestion
                .withMode(CreateMode.EPHEMERAL_SEQUENTIAL)
```
I have verified that all you have to do is change it, and the tests pass 😄

Reply:
Thank you so much @szczygiel-m 👏 
Let me apply that change